app/models/user.rb

```diff
@@ -445,8 +445,6 @@ def erase!
   end
 
   def anonymise!
-    return if info_requests.none?
```

Comment:
This change is breaking some specs, in fact I'm not sure we need this commit at all. Was there a reason why this is needed I'm not seeing?

Reply:
Removed - see comments below